portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,155 @@
+
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>hey, it's malachi 🚀</title>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css"> -->
```

Comment:
nit: I think it's good practice to avoid leaving commented out things in commits. You can always add them back!

Reply:
Done!